lxd/storage/drivers/driver_ceph.go

```diff
@@ -195,6 +195,20 @@ func (d *ceph) Create() error {
 
 		revert.Add(func() { _ = d.osdDeletePool() })
 
+		_, err = shared.TryRunCommand("ceph",
```

Comment:
Now, is this a behavioral change, what was happening before to osd pools when we didn't explicitly set the osd pool size using the default size (if not specified)?

Reply:
can we specify the pool size as part of the creation command rather than use two commands btw?

Reply:
Before, OSD pools were given the same pool size as the default OSD pool size. This is viewable by checking the OSD pool config after pool creation. So assuming a user did not explicitly set a value for the new config key, the `osd pool set` call should not result in any change to the osd pool.

Looking at the [ceph docs](https://teams.microsoft.com/l/meetup-join/19%3ameeting_M2YyMzE5OTMtY2UzNy00MjFjLWI2NTctZmMyM2U5NTM5NGQy%40thread.v2/0?context=%7b%22Tid%22%3a%22a9a572ee-9e57-46ea-b37b-b020adfd669a%22%2c%22Oid%22%3a%221170be4e-da29-46b4-9477-7628dc06a1d4%22%7d), it doesn't appear to me that there is a way to set many config options when creating a pool

Reply:
Before, OSD pools were given the same pool size as the default OSD pool size. This is viewable by checking the OSD pool config after pool creation. So assuming a user did not explicitly set a value for the new config key, the `osd pool set` call should not result in any change to the osd pool.

Looking at the [ceph docs](https://docs.ceph.com/en/reef/rados/operations/pools/#creating-a-pool), it doesn't appear to me that there is a way to set many config options when creating a pool

Reply:
> Before, OSD pools were given the same pool size as the default OSD pool size. This is viewable by checking the OSD pool config after pool creation. So assuming a user did not explicitly set a value for the new config key, the `osd pool set` call should not result in any change to the osd pool.

So no behavioral change compared with inheriting the default before?

Reply:
No change if the new config has not been explicitly included in the request.